core/commands/diff.py

```diff
@@ -18,6 +19,17 @@
 from ...common import util
 
 
+if False:
```

Comment:
What's this for?

Reply:
That's for 'mypy' typing. mypy will read and import the stuff in the 'False' branch but Sublime's python will not execute this code. 

Another way to write this is 

```
MYPY = False
if MYPY:
    ...from typing ...
```

which is probably just as confusing. @asfaltboy already asked the same question. :grin: but we first have to decide if these type annotations are welcome here at all.